src/components/display/DisplayImage.js

```diff
@@ -2,42 +2,37 @@ import axiosInstance from "../api/axios";
 import { useState, useEffect } from "react";
 import "./DisplayImage.css";
 
+let perPage = 0;
+
 const DisplayImage = ({ images, setImages }) => {
- 
-  // const [showMore, setShowMore] = useState(5);
+
+  const showImage = async () => {
+    try {
+      const res = await axiosInstance.get(`?&offset=${perPage}`);
+      setImages(res.data.results);
+    } catch (error) {
+      console.log(error);
+    }
+    perPage += 5;
+  };
 
   useEffect(() => {
-    const showImage = async () => {
-      try {
-        const res = await axiosInstance.get();
-        setImages(res.data.results);
-        console.log(res);
-      } catch (error) {
-        console.log(error);
-      }
-    };
     showImage();
   }, []);
 
+  const loadMore = () => {
+    showImage();
+  };
+
```

Comment:
I believe your `showImage` in one `useEffect` with the right amount of dependencies can handle loading the initial images and loading more images.

Hints
1. `perPage` should be a state variable
2. The `useEffect` deps should contain the `perPage` state variable
3. Your `loadMore` function should update the `perPage` state variable

Reply:
ok I will adjust this.